docs/reference/objects/accommodation/availabilty_day.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Availability Day
+parent: Accommodation
+---
+
+When using an `availability_day` object you have access to the following attributes.
+
+# availability_day.available
+
+Returns true if the Accommodation is marked as available for a day.
```

Comment:
Can we lowercase "accommodation" and perhaps reword this to "is marked as available for this date"